e2e/test/iothub/service/IoTHubServiceProxyE2ETests.cs

```diff
@@ -39,7 +40,7 @@ public async Task ServiceClient_Message_SendSingleMessage_WithProxy()
         [LoggedTestMethod]
         public async Task RegistryManager_AddAndRemoveDevice_WithProxy()
         {
-            var httpTransportSettings = new HttpTransportSettings();
+            var httpTransportSettings = new HttpTransportSettings2();
```

Comment:
Why 2?

Reply:
I'm trying to leave the existing service client code (job client, twin client, direct method client) alone for now. Eventually, I'll delete the old HTTP code and rename these HTTP2 classes to HTTP

Reply:
Similarly, the old RegistryManager class still exists since it holds twin and configurations operations that are out of scope for this first PR